Renderer.js

```diff
@@ -0,0 +1,63 @@
+
+class Renderer {
+    constructor(data) {
+        this.data = data
+    }
+
+    renderuser = () => {
+        const source = $("#user-template").html()
+        const template = Handlebars.compile(source)
+        const someHtml = template({
+            firstName: this.data.user.firstName,
+            lastName: this.data.user.lastName,
+            imguser: this.data.user.imgUser,
+            city: this.data.user.city,
+            country: this.data.user.country
+        })
+        $(".user-container").append(someHtml)
+    }
+
+    renderquote = () => {
+        const source = $("#quote-template").html()
+        const template = Handlebars.compile(source)
+        const someHtml = template({
+            quote: this.data.quote.say
+        })
+        $(".quote-container").append(someHtml)
+    }
+
+    renderpokemon = () => {
+        const source = $("#pokemon-template").html()
+        const template = Handlebars.compile(source)
+        const someHtml = template({
+            pokemonImg: this.data.pokemon.pokemonImg,
+            pokemonName: this.data.pokemon.pokemonName
+        })
+        $(".pokemon-container").append(someHtml)
+    }
+    rendermeatext = () => {
+        const source = $("#meat-template").html()
+        const template = Handlebars.compile(source)
+        const someHtml = template({
+            meatText: this.data.aboutme
+        })
+        $(".meat-container").append(someHtml)
+    }
+    renderfriends = () => {
+        const source = $("#friends-template").html()
+        const template = Handlebars.compile(source)
+        const someHtml = template({
```

Comment:
if you use {{#each friend}} in the html handlebars template you can instead send it an array so you do not need to hard code this new object